src/agent/rustjail/src/sync.rs

```diff
@@ -14,8 +14,8 @@ pub const SYNC_SUCCESS: i32 = 1;
 pub const SYNC_FAILED: i32 = 2;
 pub const SYNC_DATA: i32 = 3;
 
-const DATA_SIZE: usize = 100;
-const MSG_SIZE: usize = mem::size_of::<i32>();
+pub const DATA_SIZE: usize = 100;
```

Comment:
Should this file to be delete? since you have move all the functions to `sync_with_async.rs`

Reply:
no, the child process is still be sync.